src/components/DailyReportForm/index.js

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+import { Formik } from 'formik';
+import gql from 'graphql-tag';
+import isEmpty from 'lodash/isEmpty';
+import * as Yup from 'yup';
+import { Mutation } from 'react-apollo';
+import TextArea from '../TextArea';
+import TextInput from '../TextInput';
+import AsyncButton from '../AsyncButton';
+import IssueSelect from '../IssueSelect';
+
+const CREATE_DAILY_REPORT_MUTATION = gql`
+
+  mutation CREATE_DAILY_REPORT_MUTATION (
+    $title: String!
+    $achievement: String
+    $issues: [IssueInput!]
+    $plan: String!
+    $description: String
+    $comment: String
+  ) {
+    createDailyReport(
+      title: $title
+      achievement: $achievement
+      issues: $issues
+      plan: $plan
+      description: $description
+      comment: $comment
+    ) {
+      id
+    }
+  }
+`;
+
+const DailyReportSchema = Yup.object().shape({
+  title: Yup.string()
+    .required('Title is required.'),
+  achievement: Yup.string()
+    .required('Achievement is required.'),
+  plan: Yup.string()
+    .required('Plan is required.'),
+  // issues: Yup.array()
+  //   .of(
+  //     Yup.object().shape({
+  //       name: Yup.string().required('Required.')
+  //     })
+  //   )
+});
+
+const DailyReportForm = () => (
+  <Mutation mutation={CREATE_DAILY_REPORT_MUTATION}>
+    {(createDailyReport, { loading, error }) => (
+      <div>
+        {loading && <div>Loading...</div>}
+        {error && <div>error</div>}
+
+        <Formik
+          initialValues={{
+            title: '',
+            achievement: '',
+            issues: [],
+            plan: '',
+            description: '',
+            comment: ''
+          }}
+          enableReinitialize
+          validationSchema={DailyReportSchema}
+          onSubmit={async (values, { setSubmitting, setStatus, setErrors, resetForm }) => {
+            const { title, achievement, plan, description, comment } = values;
+
+            let issues = [];
+            if (isEmpty(values.issues)) {
```

Comment:
better 
```
if(!isEmpty(values.issues)) {
  issues = values.issues.map(issue => ({ id: issue }));
}
```

or `const issues = isEmpty(values.issues) ? [] : values.issues.map(issue => ({ id: issue }));`

Reply:
@gianglevan94 I like the second way, thanks